### DISCUSSION TITLE
Setting offsets for number or articles to skip in a row of dynamic content.

### DISCUSSION THREAD

**Discussion by srcarney** [templaza/astroid-framework#936] [Ideas]
I just made a comment on this on one of your news postings but realize I should have made this comment here instead:

I am looking forward to Astroid in 2025. In particular it says that you are adding Dynamic Content. That is something I really want and need. One feature I want to ask you to consider putting into your dynamic content is the ability to set an offset (a number value for the amount of articles that should be passed by) in order for me to make one row that features something like the most recent four items. Then I would create another row that might display the next 6 items. In order to do this I would want the offset on the second row to skip the first four items. etc...
Then I could possibly set up a third row with an offset of 10. This way there would be no duplicates.

**Comment by rinka88** [comment]
I don’t know how you understood, but for me it’s like Yootheme pro )
https://youtu.be/U1aomq1VLI0?si=v8IcNvAYU1Dj-8fW

**Reply by srcarney** [reply]
I watched that video and it was impressive. However, I did not see anything about setting offsets. A good example of this is the Raxo.org module maker.
